p2p/src/simulated/metrics.rs

```diff
@@ -0,0 +1,60 @@
+use commonware_cryptography::{utils::hex, PublicKey};
+use prometheus_client::encoding::EncodeLabelSet;
+
+#[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
+pub struct ReceivedMessage {
+    pub peer: String,
+    pub message: i32,
+}
+
+impl ReceivedMessage {
+    const UNKNOWN_TYPE: i32 = -1;
```

Comment:
In this PR, you'll only use channel here (and probably don't need any consts). 

See my comment for why that wasn't clear how to do (because I didn't implement it lol).

Reply:
I don't think including a metric for functions that return an error is worth it. Can you remove `TOO_LARGE_TYPE` and `fn new_too_large`?